ebos/eclwriter.hh

```diff
@@ -435,6 +436,41 @@ private:
         OPM_END_PARALLEL_TRY_CATCH("EclWriter::prepareLocalCellData() failed: ", simulator_.vanguard().grid().comm())
     }
 
+    void captureLocalFluxData()
+    {
+        const auto& gridView = this->simulator_.vanguard().gridView();
```

Comment:
Not sure how costly this is. Maybe should check first whether something needs to be done (multiple regions).

Reply:
> Not sure how costly this is. Maybe should check first whether something needs to be done (multiple regions).

I'll run some benchmarks to assess the computational impact of running this pass unconditionally.

Reply:
This is executed for Norne, right. Hence according to ytelses it's not a real problem.

Reply:
> This is executed for Norne, right.

Kind of.  We definitely do run the grid traversal loop to calculate the flux values, but we don't really accumulate inter-region flow rates.  While the input file does nominally request [inter-region flow summary vectors](https://github.com/OPM/opm-tests/blob/20bf08cccb001df32479a5845ef9fd1b229acfd2/norne/INCLUDE/SUMMARY/summary.data#L74-L77) it fails to specify any region ID pairs.  We don't (currently) interpret missing region ID pairs as "all possible region ID pairs", but maybe we should.